validator/src/main.rs

```diff
@@ -1574,7 +1576,8 @@ pub fn main() {
         )
         .arg(
             Arg::with_name("no_untrusted_rpc")
-                .long("no-untrusted-rpc")
+                .alias("no-untrusted-rpc")
+                .long("no-nonarchetype-rpc")
                 .takes_value(false)
                 .help("Use the RPC service of trusted validators only")
```

Comment:
"trusted" 😱

Reply:
I guess it's obvious that I made most of these changes with `sed` now :sweat_smile: